src/Scheduler.py

```diff
@@ -41,11 +52,24 @@ def cancel_scheduled_msg(self, scheduled_time_str, contact):
         else:
             print("Failed to cancel msg to contact= {0} at time= {1}".format(contact, scheduled_time_str))
 
+    def init_barcode(self, chrome_path_driver):
+        options = webdriver.ChromeOptions()
+        user_dir = os.path.join(os.path.expandvars('%LocalAppData%'),
+                                'Google', 'Chrome', 'User Data', 'Default')
+        options.add_argument(
+            f'user-data-dir={user_dir}')
+        # Be careful with pushing the session to Github!!
+        driver = webdriver.Chrome(executable_path=chrome_path_driver, chrome_options=options)
+
+        driver.get(WHATSAPP_URL)
+        WebDriverWait(driver, 60).until(
+            EC.presence_of_element_located((By.XPATH, "//*[@id=\"app\"]/div/div/div[3]")))
+        driver.quit()
+
 
 if __name__ == '__main__':
```

Comment:
I suggest moving the tests fron the source code into test code in a separate folder.

Reply:
Done